src/App.vue

```diff
@@ -17,6 +17,11 @@ export default {
         Header,
         SubHeading,
         Footer
+    },
+    computed: {
+        isNominate() { 
+            return this.$route.name === 'nominate' ? true : false;
+        }
```

Comment:
You don't need this computer property, you have a router

Reply:
Done .. .